airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -840,11 +840,6 @@ private JsonNode getDbConfig(PostgreSQLContainer psql, boolean hiddenPassword, b
       if (withSchema) {
         dbConfig.put("schema", "public");
       }
-
-      if (withNormalization) {
```

Comment:
we still want to run these tests with normalization turned on and off. similar comment as to the one in `DestinationAcceptanceTest`